lib/rules/multiline-ternary.js

```diff
@@ -18,10 +18,15 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-        schema: []
+        schema: [
+            {
+                enum: ["always", "never"]
+            },
```

Comment:
So I looked into this a little bit, and I actually think that's only for objects where a `property` key is listed.

From https://spacetelescope.github.io/understanding-json-schema/reference/object.html#properties:
>The additionalProperties keyword is used to control the handling of extra stuff, that is, properties whose names are not listed in the properties keyword. By default any additional properties are allowed.

I also tested locally, and it doesn't look like it allows it:
![screen shot 2016-08-16 at 3 52 14 pm](https://cloud.githubusercontent.com/assets/7041728/17713552/86aa4b6e-63c9-11e6-941b-ca6ca26134d8.png)

So maybe we don't need to change them all?